.github/workflows/conventional-commit-pr-title-reminder.yml

```diff
@@ -5,7 +5,7 @@ on:
     branches:
       - master
     paths:
-      - enterprise/frontend/src/embedding-sdk-bundle/**
```

Comment:
Do we still want to have some sort of warning for the `embedding-sdk-bundle` and the `sdk` scope still? As this change will only warn people about the change to packages and not the bundle.

Reply:
We need conventional commits for SDK package ONLY because SDK package changelog generation filters commits via grep and to be more percise the `(sdk)` (and now `(sdk-package)`) match is used. 

If we decide to change the changelog generation mechanism/logic or fill it manually at all, we don't need conventional commits naming at all.

Reply:
We need to request a conventional commit title if SDK Package files are changed, no matter if the PR contains some other changes or not.